src/sentry/integrations/vsts/integration.py

```diff
@@ -582,7 +582,8 @@ def create_subscription(
                 raise IntegrationProviderError(
                     "Sentry cannot communicate with this Azure DevOps organization.\n"
                     "Please ensure third-party app access via OAuth is enabled \n"
-                    "in the organization's security policy."
+                    "in the organization's security policy. \n"
+                    "The user installing the integration must have project administrator permissions."
```

Comment:
Do you want to add the org admin to it as well? seemed like this specific person had project admin.